browser/scripts/threesixty.js

```diff
@@ -535,4 +528,4 @@ E2.ui.ui360 = new function() {
 	}
 }
 
-document.addEventListener('DOMContentLoaded', E2.ui.ui360.init.bind(E2.ui.ui360))
\ No newline at end of file
+$(window).on('playerUiReady', E2.ui.ui360.init.bind(E2.ui.ui360))
```

Comment:
I think most custom events we emit are on the document, so for the sake of simplicity and having to remember only one, would like to stick to document?